src/Pages/Wizard/Wizard.js

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import './Wizard.scss';
+
+import StepOne from '../../Components/StepOne/StepOne'
+import StepTwo from '../../Components/StepTwo/StepTwo'
+import StepThree from '../../Components/StepThree/StepThree'
+
+const Wizard = () => {
+
+    const [step, setStep] = useState(1)
+
+    const nextStep = () => {
+        setStep(step + 1)
+    }
+
+    const prevStep = () => {
+        setStep(step - 1)
+    }
+
+    const switchSections = () => {
+        switch (step) {
+            case 1: 
+                return (<StepOne nextStep={nextStep} />)
+            case 2:
+                return (<StepTwo nextStep={nextStep} prevStep={prevStep}/>)
+            case 3:
+                return (<StepThree nextStep={nextStep} prevStep={prevStep}/>)
+            default:
+               return (console.log('Successful'))
```

Comment:
Leaving console logs in the code is considered a bad practice. since they can reveal to the end user something, and in bigger apps multiple console logs are gonna cause performance issues. For debugging purposes please see [this link ](https://code.visualstudio.com/docs/nodejs/reactjs-tutorial)